tst/testinstall/knuthbendix.tst

```diff
@@ -0,0 +1,95 @@
+gap> START_TEST("knuthbendix.tst");
```

Comment:
Since this only contains tests for code in `kbsemi.gi`, perhaps `kbsemi.tst` would be the more appropriate filename?